.github/workflows/clang-tidy.yml

```diff
@@ -58,6 +58,12 @@ jobs:
         run: |
           run-clang-tidy-17 -p build -j ${{ steps.number_of_threads.outputs.threads_number }} -fix -quiet 1>output.txt
 
+      - name: Run pre-commit hook
```

Comment:
Just to check: this would not end up as some infinite loop if error can't be fixed, right?

Reply:
No, it is one time formatting after fixes.  The hook only reorganises code but doesn't change it. So there should be no conflicts between the hook and clang-tidy.